modules/test/playwright/pages/object-web/object-layout/ViewObjectLayoutsPage.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {FrameLocator, Locator, Page} from '@playwright/test';
+
+export class ViewObjectLayoutsPage {
```

Comment:
Hey @vngoliveira, we can use the pattern already applied in another similar pages, like `ObjectValidationsPage` and `ObjectFieldsPage`. Therefore we don't need use the `View` prefix.